docs/security/custom-roles.md

```diff
@@ -0,0 +1,38 @@
+---
+Description: Custom roles for Semaphore platform.
+---
+
+# Custom Roles
+
+If your organization needs more roles where permissions would be assigned with 
+higher granularity, you can define custom roles that will be visible only 
```

Comment:
Maybe we can skip the `that will be visible only within your organization.` part? It should be obvious :)

Reply:
Done